packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -64,17 +65,23 @@ const AppConfig = ({children, locals = {}}) => {
             proxy={`${appOrigin}${commerceApiConfig.proxyPath}`}
             headers={headers}
         >
-            <MultiSiteProvider site={locals.site} locale={locals.locale} buildUrl={locals.buildUrl}>
-                <_CommerceAPIProvider value={locals.api}>
-                    <CustomerProvider value={{customer, setCustomer}}>
-                        <BasketProvider value={{basket, setBasket}}>
-                            <CustomerProductListsProvider>
-                                <ChakraProvider theme={theme}>{children}</ChakraProvider>
-                            </CustomerProductListsProvider>
-                        </BasketProvider>
-                    </CustomerProvider>
-                </_CommerceAPIProvider>
-            </MultiSiteProvider>
+            <CustomerProvider>
+                <MultiSiteProvider
```

Comment:
is there a particular reason why `CustomerProvider` is the parent of the `MultiSiteProvider`? I'd imagine customer data is associate with a site on the backend, so i feel like it should be the other way around, `MultiSiteProvider` being the parent.

Reply:
Good catch, it should be inside MultiSiteProvider